s3.tf

```diff
@@ -35,6 +35,21 @@ resource "aws_s3_bucket" "this" {
   versioning {
     enabled = var.s3_versioning_enabled
   }
+
+  dynamic "lifecycle_rule" {
+    for_each = var.s3_lifecycle_rules
+    iterator = rule
+
+    content {
+      id      = rule.value.id
+      enabled = rule.value.enabled
+      prefix  = rule.value.prefix
+      expiration {
+        days = rule.value.expiration_days
```

Comment:
This doesn't match the example

Reply:
This is a simpliest way. Any suggestion for alternative?

Reply:
I was referring to your provided example: https://github.com/Flaconi/terraform-aws-microservice/pull/22/files#diff-ea6693f8c9ac2ab413a0b523bb824f5eR24

```hcl
    {
      id      = "tmp"
      enabled = true
      prefix  = "tmp/"
      expiration_days = 1
    }
```

Reply:
Why is one `expiration_days` and the other `expiration.days`?